_posts/2017/05/2017-05-04-windows-installer.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Plans for Windows Installer"
+date: 2017-05-04
+time: "16:00:00"
+category: ["Community","Tools","Teaching"]
+---
+
```

Comment:
I added it at https://github.com/swcarpentry/website/pull/707/files#diff-f61ccb4a4bbbacc64b1655cf96a4d300R42.